ROADMAP.md

```diff
@@ -2,60 +2,47 @@
 
 ## About this Document
 
-This document provides an overview of the major themes driving Tanzu Community Edition development, as well as some of the features and
-capabilities that will manifest those priorities, grouped into 0-3 month, 3-6 month, and future buckets. Most items are gathered from
-the community or include a feedback loop with the community. This should serve as a reference point for Tanzu Community Edition users
-to help us prioritize and to supply input on unmet needs, and contributors to understand where the project is heading, and help
-contributors to determine if a contribution could be conflicting with a longer term plan.
+This document provides an overview of the major themes driving Tanzu Community Edition development, as well as constituent features and capabilities. Roadmap items are tracked by their implementation state within the enhancements proposal process: Awaiting Proposal, Proposal in Review, and Implementation in Progress. Most items are gathered from the community or include a feedback loop with the community.
 
-More detail about short-term delivery plans can be found in the [milestones](https://github.com/vmware-tanzu/community-edition/milestones) in
-the [GitHub project repo](https://github.com/vmware-tanzu/community-edition).
+This roadmap should serve as a reference point for Tanzu Community Edition users to help us prioritize existing features and provide input on unmet needs, and for contributors to understand where the project is heading. A roadmap also helps our contributor community to determine if a proposed enhancement conflicts with the long-term plan.
```

Comment:
>  A roadmap also helps our contributor community to determine if a proposed enhancement conflicts with the long-term plan.

I can't make sense of what this means.

Reply:
If we said we plan to do X, if a contributor is thinking they want to contribute opposite-of-X, the roadmap will tell them it is un likely to be accepted.

Reply:
New working of:

```
Contributors can also consult the roadmap when thinking of proposing new ideas, to determine if their idea conflicts with the roadmap; in that case, the team and community will need to determine whether to adjust the roadmap or to recommend changes to the idea.
```

seems far more clear. Thanks for resolving this!